packages/api/lambdas/publish-reports.js

```diff
@@ -135,10 +47,7 @@ async function handlePdrMessage(eventMessage) {
  * @returns {Promise}
  */
 function publishReportSnsMessages(eventMessage) {
```

Comment:
Can we get rid of the `publishReportSnsMessages` function and just have the handler call `handlePdrMessage` directly?

Reply:
I didn't bother because this whole thing is going to get deleted in Menno's PR, but I can change it now if you prefer

Reply:
Don't bother.